pos_lot_selection/models/stock_production_lot.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class ProductionLot(models.Model):
+    _inherit = "stock.production.lot"
+
+    available_on_pos = fields.Boolean(compute="_compute_available_on_pos", store=True)
```

Comment:
I guess we could get rid of this class by modifying the search from the POS to use the `product_qty` field? cf suggestion below.